packages/manager/apps/sign-up/src/form/form.controller.js

```diff
@@ -110,9 +111,12 @@ export default class SignUpFormAppCtrl {
     this.saveError = null;
     this.loading.init = true;
 
-    const { ovhSubsidiary } = this.$location.search();
-    if (ovhSubsidiary && ovhSubsidiary.match(/^[\w]{2}$/)) {
-      localStorage.setItem(OVH_SUBSIDIARY_ITEM_NAME, ovhSubsidiary);
+    this.$location.search();
```

Comment:
I don't understand the need for this

Reply:
As shared with you yesterday, either you keep the retrieval of subsidiary using `this.$location.search()` or you use `coreConfig`.
Since we use `this.$location.search()` to setItem in the localStorage, I would suggest to keep the code as it was initially and add `this.isIndianSubsidiary = ovhSubsidiary === SUBSIDIARIES.IN;`